.nvmrc

```diff
@@ -0,0 +1 @@
+v16.14.2
```

Comment:
should we just do 16.14? that way it will default to the latest patch version

or we could just specify `lts` instead?

Reply:
I think `16.14` makes sense. 
`lts` implies that this is _always_ compatible with the `lts` release but that isn't a guarantee we can make.